src/screens/LoadHistoryScreen.js

```diff
@@ -113,6 +113,7 @@ class LoadHistoryScreen extends React.Component {
     }
     hathorLib.WebSocketHandler.on('addresses_loaded', this.addressesLoadedUpdate);
     this.cleanData();
+    hathorLib.storage.setItem('wallet:network', NETWORK);
```

Comment:
I think we shouldn't set it in the `LoadHistoryScreen`. Maybe this can be verified and set in a single place. It must also be set in the last screen of the setup screens. What do you think?

Reply:
I was doing this but the line above this is doing a `cleanData`. It clears all storage data, so I can't set it before, like I was doing and, since `LoadHistoryScreen` might be called many times during the use of the app (e.g. in case we lose connection to the ws) doing only one time wouldn't work.

I agree that clearing everything might not be the perfect solution, so we should change this behaviour before but I feel it's okay to continue with this for now.

Reply:
I agree, we need a larger refactor. For now, it's ok for me.